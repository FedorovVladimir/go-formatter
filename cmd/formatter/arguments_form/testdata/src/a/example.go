package a

func main() {} // want "arguments form"
