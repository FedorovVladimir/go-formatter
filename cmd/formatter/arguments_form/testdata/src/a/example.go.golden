package a

func main(ctx context.Context, a int) {}
